sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_analysis_client.py

```diff
@@ -110,7 +110,10 @@ def begin_analyze_document(
         continuation_token = kwargs.pop("continuation_token", None)
 
         if continuation_token is not None:
-            return self._client.begin_analyze_document(  # type: ignore
+            _client_op_path = self._client
+            if self._api_version == DocumentAnalysisApiVersion.V2023_02_28_PREVIEW:
+                _client_op_path = _client_op_path.document_models
+            return _client_op_path.begin_analyze_document(  # type: ignore
```

Comment:
If there are any new keyword arguments added in 2023-02-28-preview, and a user tries to pass them while targeting 2022-08-31 they will cause an unexpected keyword argument with requests. It might be nice to do some client side validation in this case.

Reply:
Yeah I actually already added the param validation in the other PR where I'm exposing the new features :)